### DISCUSSION TITLE
Multipass for non-plus plans

### DISCUSSION THREAD

**Discussion by DennisKraaijeveld** [Shopify/storefront-api-feedback#42] [Ideas & Requests]
With more and more shops going headless, we really need the ability to moving over authenticated users from the custom storefront to the Shopify Checkout page. If I may ask, what is the reason to only enable this for Plus accounts? Or can we exclude this feature because it will never be available for regular plans who can't pay 2k a month

**Comment by arobbins** [comment]
I'd like to know more about this as well.

**Comment by DennisKraaijeveld** [comment]
And it not possible, it would be nice if someone from the team could explain this, so we don't have to wait and hope for a proper solution haha

**Comment by EricMcRay** [comment]
I am developing custom storefronts for my clients. They are mostly mid-size businesses. Most of them want/need multipass but I can't convince them to get Plus because It's too expensive for them. But I am sure they are ok to pay few more bucks for Multipass.
